### DISCUSSION TITLE
Option for link recipients to add comments to the shared content?

### DISCUSSION THREAD

**Discussion by RBW1999** [alangrainger/share-note#93] [Vote for upcoming features]
Usecase:

The plugin already provides a one way street for sharing content with users that don't use Obsidian. This feature would create a two-way street information exchange, where the link recipients could add their input to the shared site.

This probably can't and should not update the instance of the file in obsidian. But comments from anyone could be made visible on the shared site and thereby received by the creator.

**Comment by alangrainger** [comment]
Sorry, it's a great suggestion but that won't be in scope for this plugin.

**Comment by huyz** [comment]
You might want to check out https://web.hypothes.is/ which works with any web page
